app/templates/partials/jobs/notifications.html

```diff
@@ -34,7 +34,7 @@
           </p>
         {% elif notifications %}
           <p class="{% if job.template_type != 'letter' %}bottom-gutter{% endif %}">
-            <a href="{{ download_link }}" download class="govuk-link govuk-link--no-visited-state heading-small">Download this report</a>
+            <a href="{{ download_link }}" download class="govuk-link govuk-link--no-visited-state heading-small">Download this report (csv)</a>
```

Comment:
It’s an acronym (technically an initialism) so should be all caps:
```suggestion
            <a href="{{ download_link }}" download class="govuk-link govuk-link--no-visited-state heading-small">Download this report (CSV)</a>
```

You could also use add an `<abbr>` tag:
```suggestion
            <a href="{{ download_link }}" download class="govuk-link govuk-link--no-visited-state heading-small">Download this report (<abbr title="Comma separated values">CSV</abbr>)</a>
```
…although I’m not sure how much help ‘comma separated values’ or ’portable document format’ are to someone who doesn’t know what CSV or PDF mean:

Reply:
I've changed `csv` to `CSV` everywhere, and have added the `<abbr>` tag to the pages that members of the public see, though didn't bother adding `<abbr>` to the platform admin report download buttons.